drivers/storage/portworx/manifest/remote_test.go

```diff
@@ -8,11 +8,41 @@ import (
 	"testing"
 
 	version "github.com/hashicorp/go-version"
+	"github.com/libopenstorage/operator/drivers/storage/portworx/util"
 	corev1 "github.com/libopenstorage/operator/pkg/apis/core/v1"
 	"github.com/stretchr/testify/require"
 	"k8s.io/api/core/v1"
 )
 
+func TestRemoteManifestWithProxy(t *testing.T) {
+	// Do not run the test by default. Change httpProxy below to a working proxy.
```

Comment:
For unit tests, we want it to run on any environment. Let's test the proxy only in functional tests. 
For this unit test, see if you can test the behavior. May be check if http client is called with the proxy. 
If it is getting too complicated, we can skip the unit test.

Reply:
this unit test works and talks to real proxy, but I just dont enable it by default as the proxy is setup manually.

Reply:
I see, so you are keeping it to test this manually only. We won't be running it any environment, correct?

Reply:
Yes